src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,8 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        StringBuilder sb = new StringBuilder();
+        String editedStr = text.replaceAll("\\W", "");
+        return editedStr.equalsIgnoreCase(sb.append(editedStr).reverse().toString());
```

Comment:
instead of append you can pass editedStr inside of stringBuilder constructor

Reply:
done